src/core/ext/filters/client_channel/lb_policy/priority/priority.cc

```diff
@@ -537,17 +537,19 @@ void PriorityLb::ChildPriority::DeactivationTimer::Orphan() {
               child_priority_->priority_policy_.get(),
               child_priority_->name_.c_str(), child_priority_.get());
     }
-    if (!child_priority_->priority_policy_->channel_control_helper()
-             ->GetEventEngine()
-             ->Cancel(*timer_handle_)) {
-      timer_handle_.reset();
-    }
+    auto tmp_handle = *timer_handle_;
```

Comment:
I don't think we need this temp variable.  This code runs in the `WorkSerializer`, as does the callback, so it's impossible for the callback to run between the call to `Cancel()` and resetting `timer_handle_`.

Reply:
Good point, fixed.